### ISSUE TITLE
Choosing the activation function for the output layer

### ISSUE THREAD

**Issue by luis-gustavo** [deeplearningunb/Sign-Language-Digits-Detector#21]
## Description

I guess we should use the [softmax activation function](https://keras.io/activations/)
